hack/test-integration.sh

```diff
@@ -61,6 +61,7 @@ function buildImage {
 }
 
 function verifyForBuildSuccess {
+  sleep 300
```

Comment:
was this put on purpose? why do we need this?

Reply:
That was needed because otherwise, the integration test would expect builds to complete instantly and would fail with `build was not completed fully, the state was Running but expected to be 'Complete'`. I added this sleep statement to allow the builds time to complete